include/z64actor.h

```diff
@@ -147,7 +147,8 @@ typedef struct {
 #define ACTOR_FLAG_5 (1 << 5)
 #define ACTOR_FLAG_6 (1 << 6)
 #define ACTOR_FLAG_7 (1 << 7)
-#define ACTOR_FLAG_8 (1 << 8)
+// Player has requested to talk to the actor; Player uses this flag differently than every other actor
```

Comment:
It only now occurs to me "player uses this flag" isn't super clear that it's talking about "use" as in `player->actor.flags &/|` (instead of "use" as in set on other actors to request talking)
Maybe this could do?:
```suggestion
// Player has requested to talk to the actor; When set on the Player actor, has a specific different meaning
```

Reply:
In the same vein of https://github.com/zeldaret/oot/pull/1567#discussion_r1359478089 , Im starting to think it would be better if the name covered both the player case and actor case, to better facilitate this confusion of whos requesting/accepting what

Reply:
I different approach I've been thinking is because of the difference in two meanings, maybe have two defines for the same flag? One to be used with player and one to be used with different actors? Something like:
```c
// Signals that player has requested to talk to the actor
// Player will retain this flag until the player is finished talking
// Actor will retain this flag until `Actor_AcceptTalkRequest` is called or manually accepted by the actor
#define ACTOR_FLAG_PLAYER_TALKING (1 << 8) // For player
#define ACTOR_FLAG_TALK_REQUESTED (1 << 8) // For actor
```

Reply:
Updated the above comment a little bit. Also worth noting that there is a player-flag that also used to indicate that the player is talking, but seems to have slightly different conditions than player's use of `ACTOR_FLAG_PLAYER_TALKING` (i.e. `PLAYER_STATE1_6` -> `PLAYER_STATE1_TALKING`), so neither of those 2 player-specific flags would be named in this PR. Should be done separately with more player talk docs where better and more specific names might be found

Reply:
Kinda feel like this flag should just be named "TALK" in the short term, to cover the meanings for both regular actors and the player. And as we document players side more, can decide what exactly we want to do with the flag